ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -10,6 +10,23 @@ import UIKit
 import ThunderBasics
 import ThunderTable
 
+public struct QuizConfiguration {
+    
+    /// Shared `QuizConfiguration`
+    public static var shared = QuizConfiguration()
+    
+    /// If `true`, the `QuizQuestionViewController` `continueButton` will
+    /// enable/disable itself depending on whether the question has been answered.
+    public var requireAnswer: Bool
+    
+    /// Default public memberwise initializer
+    ///
+    /// - Parameter requireAnswer: `Bool`
+    public init (requireAnswer: Bool = true) {
```

Comment:
If we're using shared, should this be `private`? :) Should be able to :`QuizConfiguration.shared.requireAnswer = true` right? :)

Reply:
Just means subclasses can set it via one of (their choice)
```
var quizConfiguration = QuizConfiguration.shared
quizConfiguration.requireAnswer = false
QuizConfiguration.shared = quizConfiguration
```
or
```
QuizConfiguration.shared = QuizConfiguration(requireAnswer: false)
```

Reply:
Or what you said! ;)